enhancements/kube-apiserver/audit-policy.md

```diff
@@ -104,74 +114,86 @@ depth using the new API.
 
 ## Proposal
 
-We propose to add an `audit.profile` field to `apiservers.config.openshift.io/v1` which selects the
-audit policy to be deployed to all OpenShift-provided API servers in the cluster. From the beginning
-we provide the following profiles:
+We propose to add an `audit` struct to `apiservers.config.openshift.io/v1` with
+1. a `customRules` list for custom profiles matching requests by a group the requesting user is part of.
+2. a `profile` field for requests that don't match any rule in (1).
+This struct specifies the audit policy to be deployed to all OpenShift-provided API servers in the cluster.
 
-- `Default`: this is the existing default policy. `Default` is also the default value in the `APIServer` custom resource:
+The generated `audit.k8s.io/v1beta1` has a constant preamble:
 
```

Comment:
Also will need to specify that it will go to `all users` if a group isn't inputted by the user

Reply:
yes. If you don't specify the group, it applies to all. By putting the all-groups rules at the end, they act as fallback when no per-group rule applies.